lib/routes/email.js

```diff
@@ -14,7 +14,7 @@ const AUTH_SERVER_URL = config.get('authServer.url') + '/account/profile';
 module.exports = {
   auth: {
     strategy: 'oauth',
-    scope: ['profile', 'profile:email']
+    scope: ['profile', 'profile:email', /* openid-connect scope */'email' ]
```

Comment:
I suspect we'll have to also add this logic in the auth-server `/account/profile` endpoint, otherwise we won't be able to read it out in a non-mocked setting.